### DISCUSSION TITLE
Lentil-BHGC Roadmap

### DISCUSSION THREAD

**Discussion by BrianHanke** [BrianHanke/lentil-bhgc#2] [Announcements]
Just wanted to provide a mini-roadmap of the things I'd like to add to Lentil-BHGC:

- Keep Lentil builds up to date in step with Arnold SDK.
- Windows installer to simplify setup.
- Maya shelf button to connect Lentil imager and operator with one click.
- Continue to refine parameter interface and tooltips for clarity.
- Sample images showing what each parameter does.
- Houdini Solaris support (long term goal, need improved Arnold Solaris support first).

**Comment by amaanakram** [comment]
Hi Brian. Thanks for maintaining Lentil. 
Do you have any plans to support Arnold's API from 7.3.1.0 onwards? Specifically, the changes needed to support he Imager API? thank you

**Reply by BrianHanke** [reply]
I updated the commercial builds for 7.3.0.0 recently and in my limited testing it appears to work fine with 7.3.1.0+. I was worried that the new Imager API would break Lentil, but the "imager" in Lentil is actually implemented as a driver in the code. There might be some benefits to using the new API, but I don't think it's necessary to change, plus it would be a ton of work.

**Reply by amaanakram** [reply]
Thanks Brian. I'll have a look at how it fits in with HtoA for us.
